### ISSUE TITLE
parameter

### ISSUE THREAD

**Issue by CHENHUASHAN003** [icc2115/Neural-GC#18]
Hello author,

I'm sorry to bother you. I'm a college student from China. Currently, I'm trying to reproduce the time series examples in your paper. Could you please provide the parameters for using CLSTM to detect the VAR causal relationship and the parameters for using CMLP to detect the Lorenz96 causal relationship? Also, I'd like to ask how you selected the hyperparameters in the examples you provided, as well as your thought process and steps for selecting them?

Thank you!

**Comment by iancovert** [comment]
Hi, for the cLSTM and cMLP parameters do you mean the architecture? The paper mentions that we used one hidden layer with width 100 for those simulated datasets. We didn't tune these very carefully because there isn't a great validation setup. 

For the penalty strength, we used a range of values to trace precision-recall curves for each dataset (see Fig 7). I've recommended that other people tune this based on the validation loss if they really need to choose a single value, but I think tracing the curve for different penalty strengths is a better way to evaluate and compare GC selection methods.
